components/cards/default/card.hbs

```diff
@@ -1,4 +1,32 @@
 <div class="section--bg-white demo-section">
+  <a href="#" class="content-block--clickable">
+    <div class="content-block__upper">
+      <img
+        class="content-block__image"
+        src="https://via.placeholder.com/535x300"
+        role="presentation"
+        alt=""
+      />
+      <time 
+        class="content-block__date"
+        datetime="2021-06-10">
+          <span class="content-block__date-month" aria-label="June">Jun</span>
+          <span class="content-block__date-day">10</span>
+      </time>
+    </div><!-- .content-block__upper -->
+
+    <div class="content-block__lower">
+      <p class="content-block__subtitle">Digitalks Webinar</p>
+      <h4 class="content-block__title"><span>RSVP: This is a featured event that is topical and relevant.<span></h4>
+      <p class="content-block__text">Lorem ipsum dolor, sit amet consectetur
+        adipisicing elit. Voluptate sit id harum aliquid ea.
+      </p>
+      <div class="button--medium-primary content-block__button">Learn More</div>
```

Comment:
Resolved the nesting of `<a>` tags by switching the button to a `<div>`. This resolves the issue of having two `#href`s.